exploration/maintaining-registry.md

```diff
@@ -155,73 +169,80 @@ _What prior decisions and existing conditions limit the possible design?_
-It can also contain _options_ registered for _functions_ found in the default registry.
-Implmentations are not required to implement either _functions_ or _options_ to claim
-MF2 conformance, but MUST NOT implement functions or options that conflict with RGI entries.
-Each function describes it's operand or operands,
-its formatting options (if any) and their values,
-its selection options (if any) and their values,
-its formatting behavior (if any),
-its selection behavior (if any),
-and its resolved value behavior.
-Each option registry entry describes whether it affects formatting, selection or both;
-what its values are;
-and whether it is retained or affects the resolved value.
-Items in the RGI function or option registries are stable and subject to stability guarantees
-except that they MAY be promoted to the default registry.
+
+`Optional` functions are stable and subject to stability guarantees once they
+reach the status of **Released**.
+Implmentations are not required to implement _functions_ or _options_ with an `Optional` status
+when claiming MF2 conformance.
+Implementations MUST NOT implement functions or options that conflict with `Optional` functions or options.
+
+`Optional` values may have their status changed to `Standard`,
+but not vice-versa.
+
 > Option Examples `:datetime` might have a `timezone` option in LDML46.
 > Function Examples: We don't currently have any, but potential work here
 > might includes personal name formatting, gender-based selectors, etc.
 
-RGI includes functions that are not
-normatively required but whose names, operands, and options are recommended.
-Implementations SHOULD use these function signatures
-when implementing the described functionality.
-This will promote message interoperability
-and reduce the learning curve for developers, tools, and translators.
-
-3. **Unicode Reserved Namespace**
-This registry is for items in the namespace `u:`, which is reserved for use by the Unicode Consortium.
-This registry can contain _functions_ or _options_.
-Implementations are not required to implement any values found in this registry
-and may adopt or ignore registry entries at their discretion.
-Items in the Unicode Reserved Namespace function or option registries are stable and subject to stability guarantees.
-This registry might sometimes be used to incubate functionality before
-promotion to the RGI or default registry in a future release.
+The CLDR-TC reserves the `u:` namespace for use by the Unicode Consortium.
+This namespace can contain _functions_ or _options_.
+Implementations are not required to implement these _functions_ or _options_
+and may adopt or ignore them at their discretion,
+but are encouraged to implement these items.
+
+Items in the Unicode Reserved Namespace are stable and subject to stability guarantees.
+This namespace might sometimes be used to incubate functionality before
+promotion to the default namespace in a future release.
```

Comment:
Given that the `u:` space is stable, doesn't this mean that anything "incubated" there would need to remain there even after its "promotion"? And given that we are also proposing a process for introducing new items directly in the root namespace, why/what would we want to incubate in such a manner? I'd prefer this to be left out.

Reply:
> Given that the u: space is stable, doesn't this mean that anything "incubated" there would need to remain there even after its "promotion"?

Yes. I think I say that somewhere?

> And given that we are also proposing a process for introducing new items directly in the root namespace, why/what would we want to incubate in such a manner? I'd prefer this to be left out.

This is permission to promote something from `u:` to standard, sort of like the way browsers use prefixes in CSS. This might be a bad idea.